gensim/corpora/wikicorpus.py

```diff
@@ -110,9 +110,6 @@ def remove_markup(text):
         if old == text or iters > 2:
             break
 
-    # the following is needed to make the tokenizer see '[[socialist]]s' as a single word 'socialists'
-    # TODO is this really desirable?
```

Comment:
Indeed I don't think its desirable since it will split interlinks which should be represented by one token. I.e 
"Computer Science" fits the context better than "computer" and "science"